luaui/Widgets/gui_selectedunits_gl4.lua

```diff
@@ -119,9 +120,8 @@ local function AddPrimitiveAtUnit(unitID)
 	)
 end
 
-local drawFrame = 0
-function widget:DrawWorldPreUnit()
-	drawFrame = drawFrame + 1
+
+local function DrawSelections(selectionVBO, isAir)
```

Comment:
This isAir argument is not used, it can be removed

Reply:
might be needed, doesn't cost anything